### DISCUSSION TITLE
how can I change the font and the colors ?

### DISCUSSION THREAD

**Discussion by remilambert** [ColorlibHQ/AdminLTE#3900] [Q&A]
Hello,

I downloaded AdminLTE 3.1.0 recently and I would like to know how I can change the font used (I want to reduce globally its weight and its size) and I would like to change the colors to use less "tonic" ("flashy") colors. 
How are we supposed to do ? I tried to override some rules in CSS file but without luck...

Thanks for your help !!

R.

**Comment by REJack** [comment]
You need to customize it via SCSS, but it's not easy to reduce globally its weight/size.

Here is a demo repo how to customize AdminLTE, https://github.com/REJack/AdminLTE-Customized.
